extrafunctions/get_drive_mappings.m

```diff
@@ -0,0 +1,54 @@
+function [mappings, error_msg] = get_drive_mappings()

```

Comment:
If you foresee this function used only in aa_init, then it should be a subfunction rather than a separate one.

Reply:
When Parallel Server is used, or remote submission, this may also be needed during execution of jobs.

Also, having it as a separate function makes it possible to have a unit test for this function (not implemented yet). Unit testing of subfunctions is usually hard / impossible.